db/migrations/README

```diff
@@ -0,0 +1 @@
+Single-database configuration for Flask.
```

Comment:
🔧 Copy over information about DBs from root README at least :)

Reply:
This README is automatically generated by Flask-Migrate so don't have any control over it!